src/imagery/i.ann.mktrainset/Makefile

```diff
@@ -0,0 +1,7 @@
+MODULE_TOPDIR = ../..
+
+PGM = i.ann.mktrainset
```

Comment:
Hi @YannChemin Please consider to submit the new `i.ann.mktrainset` addon as a separate pull request.

Reply:
He is submitting PRs with the default branch, so that's what is happening. There isn't a branch for each of the PRs (and we get notified of his CI failures as that branch is involved in a PR we  are involved in)